models/federation.go

```diff
@@ -0,0 +1,18 @@
+package models
```

Comment:
rename this file to `follow` or something as `/following/*` endpoints are using the same response struct.